test/fixtures/mockRepositoriesData.json

```diff
@@ -61,6 +61,24 @@
 							"name": "develop"
 						},
 						"viewerPermission": "ADMIN"
+					},
+					{
+						"name": "tc39-ci",
+						"nameWithOwner": "name/tc39-ci",
+						"isPrivate": false,
+						"defaultBranchRef": {
+							"name": "main"
+						},
+						"viewerPermission": "ADMIN"
+					},
+					{
+						"name": "ecma262",
+						"nameWithOwner": "name/ecma262",
+						"isPrivate": false,
+						"defaultBranchRef": {
+							"name": "main"
+						},
+						"viewerPermission": "ADMIN"
```

Comment:
none of these seem to have an `isSecurityPolicyEnabled` property?

Reply:
so you mean should i add `isSecurityPolicyEnabled` in both objects?

Reply:
@ljharb 
"SecurityPolicyEnabled": true or false

Reply:
I mean that the objects should represent an actual API response, which should have that property.

Reply:
@ljharb I tried to read the extra changes that you have made in the other PR #74, and to make changes similar to it here but I'm stuck. can we schedule a meeting to finalize this PR before the end of the fellowship?

Reply:
Absolutely!

Reply:
maybe at the same time as now but tomorrow? it's now 7:15 AM can we meet at 7 at discord?

Reply:
12 hours from now works fine; ping me.